kfs-core/src/main/resources/edu/arizona/kfs/db/changelog/changesets/db.changelog-UAF-2802.xml

```diff
@@ -0,0 +1,136 @@
+    <comment>
+      UAF-2902 Adding responsibility without an assigned group for PCDM routing should go directly to final without routing
+    </comment>
+    <insert tableName="KRIM_RSP_T">
+      <column name="RSP_ID" valueComputed="KRIM_RSP_ID_S.NEXTVAL"/>
+      <column name="RSP_TMPL_ID" value="1"/>
+      <column name="NM" value="Review PCDM CentralAdministrationReview"/>
+      <column name="NMSPC_CD" value="KFS-FP"/>
+      <column name="VER_NBR" value="1"/>
+      <column name="DESC_TXT" value="Enforce a central office review of any changes to the Procurement Cardholder maintenance documents. This will not be assigned to anyone, since PCARD admin has initiate and do not want the doc to route."/>
+      <column name="OBJ_ID" value="UAF-2802-4"/>
+      <column name="ACTV_IND" value="Y"/>
+    </insert>
+    <insert tableName="KRIM_RSP_ATTR_DATA_T">
+      <column name="ATTR_DATA_ID" valueComputed="KRIM_ATTR_DATA_ID_S.NEXTVAL"/>
+      <column name="RSP_ID" valueComputed="(SELECT RSP_ID FROM KRIM_RSP_T WHERE NM='Review PCDM CentralAdministrationReview' AND OBJ_ID='UAF-2802-2')"/>
+      <column name="KIM_TYP_ID" value="7"/>
+      <column name="KIM_ATTR_DEFN_ID" value="41"/>
+      <column name="ATTR_VAL" value="false"/>
+      <column name="VER_NBR" value="1"/>
+      <column name="OBJ_ID" value="UAF-2802-5"/>
+    </insert>
+    <insert tableName="KRIM_RSP_ATTR_DATA_T">
+      <column name="ATTR_DATA_ID" valueComputed="KRIM_ATTR_DATA_ID_S.NEXTVAL"/>
+      <column name="RSP_ID" valueComputed="(SELECT RSP_ID FROM KRIM_RSP_T WHERE NM='Review PCDM CentralAdministrationReview' AND OBJ_ID='UAF-2802-2')"/>
+      <column name="KIM_TYP_ID" value="7"/>
+      <column name="KIM_ATTR_DEFN_ID" value="16"/>
+      <column name="ATTR_VAL" value="CentralAdministrationReview"/>
+      <column name="VER_NBR" value="1"/>
+      <column name="OBJ_ID" value="UAF-2802-6"/>
+    </insert>
+    <insert tableName="KRIM_RSP_ATTR_DATA_T">
+      <column name="ATTR_DATA_ID" valueComputed="KRIM_ATTR_DATA_ID_S.NEXTVAL"/>
+      <column name="RSP_ID" valueComputed="(SELECT RSP_ID FROM KRIM_RSP_T WHERE NM='Review PCDM CentralAdministrationReview' AND OBJ_ID='UAF-2802-2')"/>
+      <column name="KIM_TYP_ID" value="7"/>
+      <column name="KIM_ATTR_DEFN_ID" value="13"/>
+      <column name="ATTR_VAL" value="PCDM"/>
+      <column name="VER_NBR" value="1"/>
+      <column name="OBJ_ID" value="UAF-2802-7"/>
+    </insert>
+    <insert tableName="KRIM_RSP_ATTR_DATA_T">
+      <column name="ATTR_DATA_ID" valueComputed="KRIM_ATTR_DATA_ID_S.NEXTVAL"/>
+      <column name="RSP_ID" valueComputed="(SELECT RSP_ID FROM KRIM_RSP_T WHERE NM='Review PCDM CentralAdministrationReview' AND OBJ_ID='UAF-2802-2')"/>
+      <column name="KIM_TYP_ID" value="7"/>
+      <column name="KIM_ATTR_DEFN_ID" value="40"/>
+      <column name="ATTR_VAL" value="false"/>
+      <column name="VER_NBR" value="1"/>
+      <column name="OBJ_ID" value="UAF-2802-8"/>
+    </insert>
+
+    <rollback>
```

Comment:
For the rollback, I think you need to delete the responsibility attributes first, then the responsibility. The responsibility delete would be last.